assets/cluster-version-operator/cluster-version-operator-deployment.yaml

```diff
@@ -42,6 +42,24 @@ spec:
 {{ if .MasterPriorityClass }}
       priorityClassName: {{ .MasterPriorityClass }}
 {{ end }}
+      initContainers:
+        - name: prepare-payload
+          image: {{ .ReleaseImage }}
+          imagePullPolicy: Always
+          command:
+            - /bin/bash
+          args:
+            - -c
+            - -|
+              cp -R /manifests /var/payload/
+              cp -R /release-manifests /var/payload/
+
+              # TODO(jonesbr): This InitContainer can be removed once the annotiation is removed from these files
```

Comment:
Do you have a link to an issue or bugzilla for this TODO?

Reply:
Added links to the PRs that need to be merged and cherry-picked